<launch>
  <node pkg="qr_detector" type="qr_detector_node" name="qr_detector" output="screen">
    <param name="camera_topic" type="string" value="/camera/rgb/image_raw" />
    <param name="camera_info_topic" type="string" value="/camera/rgb/camera_info" />
    <param name="output_topic" type="string" value="/qr_decode" />
	<param name="qr_size" type="double" value="0.04" />
  </node>
</launch>